Modules/Core/Common/test/itkAnatomicalOrientationGTest.cxx

```diff
@@ -19,6 +19,9 @@
 #include <gtest/gtest.h>
 
 #include "itkGTest.h"
+#ifndef ITK_FUTURE_LEGACY_REMOVE
```

Comment:
Shouldn't this be `#ifdef`, not `#ifndef`?

Reply:
If ITK_FUTURE_LEGACY_REMOVE is true then the method with the depreciation is removed, along with the test of it.